src/components/filters/status-dropdown/status-dropdown.tsx

```diff
@@ -22,28 +22,40 @@ const statusOptions: DropdownOption[] = [
 
 export const StatusDropdown = () => {
 	const { filters, setStatusFilter } = useSokStore();
-	const value = filters.status ? mapStatusTilDropdownOption(filters.status) : null;
+	const value = filters?.status ?? undefined;
+	const selectRef = useRef<HTMLSelectElement>(null);
 
-	function handleOnStatusSelectedChanged(selectedOption: DropdownOption | null) {
-		const nyStatus = selectedOption ? (selectedOption.value as UtkastStatus) : undefined;
+	function handleOnStatusSelectedChanged(selectedOption: React.ChangeEvent<HTMLSelectElement>) {
+		const nyStatus = selectedOption.target.value as UtkastStatus;
 		setStatusFilter(nyStatus);
 	}
 
+	function handleOnNullstillButtonClicked() {
+		if (selectRef.current) {
+			selectRef.current.selectedIndex = 0;
+		}
+		setStatusFilter(undefined);
+	}
+
 	return (
-		<>
-			<label className="typo-element" htmlFor="status-filter">
-				Status
-			</label>
+		<HStack gap="2">
 			<Select
-				aria-label="Filtrer på status"
-				inputId="status-filter"
-				placeholder="Filtrer på status"
-				value={value}
-				isClearable
-				isSearchable={false}
-				options={statusOptions}
-				onChange={handleOnStatusSelectedChanged as any}
-			/>
-		</>
+				label="Statusfilter"
+				size="small"
+				value={value as string}
+				onChange={handleOnStatusSelectedChanged}
+				ref={selectRef}
+			>
+				<option value={''}>Velg status</option>
```

Comment:
En liten kuriositet, men det er mulig å velge `Velg status`-alternativet igjen etter at man først har valgt en av de faktiske statusene. Om man gjør det så får man melding "Fant ingen brukere". Jeg synes dette er litt dårlig UX, og at vi heller bør ha et "Alle"-alternativ som faktisk viser alle brukerne og ikke meldingen.

Reply:
Godt poeng! Dette skjer jo ikke lokalt. Det er ikke `setStatusFilter(undefined)` som forårsaker feilen, den gjør at nullstill-knappen fungerer. Men det serveren ikke tåler er å få en tom streng. Hadde det vært lov å sette `value` på `option`-elementet til `null` så hadde det funka, men det er da ikke gyldig. Så vi må muligens legge inn at vi skal takle tom streng 🤔